tests/integration/cases/run_macaron_sh_script_unit_test/test_run_macaron_sh.py

```diff
@@ -41,12 +40,11 @@ def test_macaron_command() -> int:
         print(f"test_macaron_command[{name}]:", end=" ")
 
         result = subprocess.run(
-            [
-                "scripts/release_scripts/run_macaron.sh",
+            [  # nosec
```

Comment:
```suggestion
            [  # nosec B603
```

Reply:
Addressed in https://github.com/oracle/macaron/pull/782/commits/f3ecbf0b72a628ae8fa762959cd967185d8044ef